### DISCUSSION TITLE
Can't use "Add To Start Menu", doesn't add to the Start Menu

### DISCUSSION THREAD

**Discussion by korishan** [valinet/ExplorerPatcher#1140] [General]
When searching for an item (Open Start and type in name of app) and then right click that item and click on "Add to Start", it doesn't show up in the Start Menu. I'm guessing it's showing up in the default W11 start menu.
I can click on any item in the Program List and drag it, or Add to Start, but if they aren't listed in the Program List, they won't add to the ExplorerPatcher Start Menu

I have several apps from the Store and I can't add them to the Start Menu, for example.

**Comment by yyqsg888** [comment]
Have the same issues. Seems to occur to all Windows Apps programs installed via Windows Store. Unable to find a workaround even by pinning the shortcut to the app.

**Comment by NethengeicWE** [comment]
So do I.It seems that you cannot find the apps installed from the windows apps programs when using ep[22621.1413.54.1].
There is a solution for this: Create a shortcut to the app and add this shotcut to the star menu,some app aren't have icon so you may need to change icon manually.If doesn't work,try again.
But after update to [22621.1413.54.2] and latest I found that althongh you can found apps which aren't appeal before,you can't add all apps to the pinned title,so does the unpinned.
By the way,the star menu isn't the same file:you can see the file via app list instead of pinned
win10: C:\Users\Username\AppData\Roaming\Microsoft\Windows\Start Menu\Programs, 
win11: C:\ProgramData\Microsoft\Windows\Start Menu\Programs,
sorry for my bad english:(

**Comment by valinet** [comment]
Properties - Start menu - Pin tiles to Windows 10 Start menu...

**Reply by korishan** [reply]
> Properties - Start menu - Pin tiles to Windows 10 Start menu...

All this does is open File Explorer for me and shows my default Explorer location (which for me is This PC)

**Reply by mdio33** [reply]
> Properties - Start menu - Pin tiles to Windows 10 Start menu...

That only work for standard windows applications, all modern apps from Windows 10 like Weather, News or even Mail, don't pin as every new app from the Store, like Skype or Whatsapp that does not pin too.
Even from "Properties - Start menu - Pin tiles to Windows 10 Start menu", if you can find modern apps from explorer, what it does is pinning a shotcut with no icon that does not work.

I am very sad about it, because i can't stand the new Win11 Menu, it's just useless, but like this, Explorer Patcher is no good too. :(

**Comment by borisBelloc** [comment]
hello, 
now whatsapp on PC force you to use the app from Microsoft store; it's impossible to pin it to start menu 😥
the workaround is fine for .exe but does not seem to work with apps from the Microsoft store.

Any idea what to do ? 
I tried to google a solution and don't find any results it sent me here

**Comment by stiffbruno** [comment]
Hello, no solution yet? thanks.

**Reply by vippium** [reply]
> Hello, no solution yet? thanks.

there is a solution that you can manually create a shortcut of that program in the folder (where all shortcuts reside) or rename the dxgi.dll in **Microsoft.Windows.StartMenuExperienceHost_cw5n1h2txyewy** in SystemApps. But issue after renaming is you can't use R-Click in SM and the closing animation of SM will be slow...If u prefer, u can use.

**Comment by pyrates999** [comment]
Is it this? https://github.com/valinet/ExplorerPatcher/issues/1108

**Comment by doneykoo** [comment]
found workarounds at: Windows 10 Start menu issues and quirks #1679

**Comment by niroshft** [comment]
![Pin-Items-to-Explorere-Patcher-Start-menu](https://github.com/user-attachments/assets/e9e7f905-7b0a-4c8f-b0e7-7062077568eb)

**Found a Working Solution** 

1) Copy actual programme's exe file to desktop (ex: vlc.exe) 
2) Create a shortcut from above exe in same location > right click > pin to start menu
3) Startmenu Shortcut will appear > right click > more > open file location
4) Right click on this > properties > change the "target" location to actual exe location 
ex : "C:\Users\ _**yourname**_\Desktop\vlc.exe" to "C:\Program Files\VideoLAN\VLC\vlc.exe"

👏👌😁
